packages/rdma-core/rdma-core.spec

```
@@ -0,0 +1,163 @@
+%global abiver rdmav34
+
+Name: %{_cross_os}rdma-core
+Version: 53.1
+Release: 1%{?dist}
+Summary: RDMA core userspace infrastructure, including core libraries and util programs.
+License: Linux-OpenIB AND MIT
+Source0: https://github.com/linux-rdma/rdma-core/releases/download/v%{version}/rdma-core-%{version}.tar.gz
+Source100: libibverbs-tmpfiles.conf
+
+# RDMA logdog configuration
+Source200: logdog.rdma.conf
+
+BuildRequires: cmake
+BuildRequires: %{_cross_os}libnl-devel
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}libnl
+
+%description
+%{summary}.
+
+%package devel
+Summary: RDMA core development libraries and headers
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n rdma-core-%{version} -p1
+
+%build
+%{cross_cmake} . \
+  -DNO_PYVERBS=1 \
+  -DNO_MAN_PAGES=1 \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX:PATH=%{_cross_prefix} \
+  -DCMAKE_INSTALL_BINDIR:PATH=%{_cross_bindir} \
+  -DCMAKE_INSTALL_SBINDIR:PATH=%{_cross_sbindir} \
+  -DCMAKE_INSTALL_SYSCONFDIR:PATH=%{_cross_sysconfdir} \
+
+%make_build
+
+%install
+%make_install
+
+install -d %{buildroot}%{_cross_tmpfilesdir}
+install -p -m 0644 %{S:100} %{buildroot}%{_cross_tmpfilesdir}/rdma-core.conf
+
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/libibverbs.d
+install -p %{buildroot}%{_cross_sysconfdir}/libibverbs.d/efa.driver %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/libibverbs.d
+
+install -d %{buildroot}%{_cross_datadir}/logdog.d
+install -p -m 0644 %{S:200} %{buildroot}%{_cross_datadir}/logdog.d
+
+%files
+%license COPYING.md COPYING.BSD_MIT ./ccan/LICENSE.MIT
```

nit, probably not worth fixing:
```suggestion
%license COPYING.md COPYING.BSD_MIT ccan/LICENSE.MIT
```